assets/docs/tulasiram58827/craft-text-detector/1.md

```diff
@@ -0,0 +1,10 @@
+### CRAFT-TEXT-DETECTOR
```

Comment:
Since this file does not specify `asset-path` I assume you don't want to publish the TF model used to extract the Lite model. In that case you should change the first line to:

`# Placeholder tulasiram58827/craft-text-detector/1`

Reply:
Updated.